pkg/apis/core/validation/shoot.go

```diff
@@ -949,6 +949,35 @@ func ValidateKubeAPIServerLogging(loggingConfig *core.KubeAPIServerLogging, fldP
 	return allErrs
 }
 
+// ValidateKubeAPIServerRequests validates the given KubeAPIServer request fields.
+func ValidateKubeAPIServerRequests(requests *core.KubeAPIServerRequests, fldPath *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+
+	if requests != nil {
+		const maxMaxNonMutatingRequestsInflight = 800
```

Comment:
Not trying to offend any Max out there, but I would still prefer to have only one Max here. How about `maxNonMutatingRequestsInflightLimit` instead.

```suggestion
		const maxNonMutatingRequestsInflightLimit = 800
```

Reply:
Technically, it's correct to have two `max` 😉 But I can change it, if you want.

Reply:
My Max stack is limited to one. Hence, I would appreciate if you prevent an overflow of my Max stack ;-) .